uui/i18n.ts

```diff
@@ -1,4 +1,7 @@
+import {i18n as uuiI18n} from '@epam/uui-core';
+
 export const i18n = {
+    ...uuiI18n,
```

Comment:
move i18n locale from uui-core to uui-components

Reply:
done